src/NumericalAlgorithms/Interpolation/InterpolationTargetWedgeSectionTorus.hpp

```diff
@@ -102,14 +102,14 @@ struct WedgeSectionTorus {
   struct UniformRadialGrid {
     using type = bool;
     static constexpr Options::String help = {
-        "Use uniform radial grid [default: LGL grid]"};
-    static type default_value() noexcept { return false; }
+        "Use uniform radial grid [suggested: LGL grid]"};
+    static type suggested_value() noexcept { return false; }
   };
   struct UniformThetaGrid {
     using type = bool;
     static constexpr Options::String help = {
-        "Use uniform theta grid [default: LGL grid]"};
-    static type default_value() noexcept { return false; }
+        "Use uniform theta grid [suggested: LGL grid]"};
+    static type suggested_value() noexcept { return false; }
```

Comment:
Here (and radial option above) also, it's not clear to me we want a suggested value — I think the use of LGL vs Uniform grid will depend on the intended use of the interpolated data (visualization vs integrating stuff on the grid, for instance), and I don't see why one choice should warn. @markscheel may have input on this.

Reply:
Agreed